### PR TITLE
Add Chinese translation for task-telemetry-fluentd

### PR DESCRIPTION
**Pull request by pigletfly**
@gyliu513 pls review

### CODE REVIEW

File: content_zh/docs/tasks/telemetry/fluntd/index.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+```xml
+<source>
+  type forward
+</source>
+```
+
+将 Mixer 连接到所有可能 Fluentd 配置的完整细节超出了此任务的范围。
+
+### Fluentd/Elasticsearch/Kibana 栈
```
Comment (gyliu513) [inline content_zh/docs/tasks/telemetry/fluntd/index.md]: s/栈/软件栈

File: content_zh/docs/tasks/telemetry/fluntd/index.md [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+title: 使用 Fluentd 记录日志
+description: This task shows you how to configure Istio to log to a Fluentd daemon
+weight: 60
+keywords: [telemetry,logging]
+---
+
+此任务将展示如何配置 Istio 创建自定义日志条目并且发送给 [Fluentd](https://www.fluentd.org/) 守护进程。
+Fluentd 是一个开源的日志收集器，支持多种[数据输出](https://www.fluentd.org/dataoutputs)并且有一个可插拔架构。
+[Elasticsearch](https://www.elastic.co/products/elasticsearch)是一个流行的后端日志记录程序，
+[Kibana](https://www.elastic.co/products/kibana) 用于查看。在任务结束后，一个新的日志流将被加载发送日志到示例 Fluentd/Elasticsearch/Kibana 栈。
```
Comment (gyliu513) [inline content_zh/docs/tasks/telemetry/fluntd/index.md]: s/栈/软件栈

**Comment by geeknoid** [comment]
Please update this PR to use the new annotations for code blocks. Instead of backticks, you need to use

{{< text bash >}}

Please see [here](https://preliminary.istio.io/about/contribute/writing-a-new-topic/#embedding-preformatted-blocks) for info

Thanks.

**Comment by geeknoid** [comment]
Please update the various code blocks to match the ones from the English version, that'll fix the analysis errors.

**Pull request closed without merging**